### ISSUE TITLE
Allow pagination at top and/or bottom of carousel

### ISSUE THREAD

**Issue by DrMeers** [SSENSE/vue-carousel#313]
**Is your feature request related to a problem?** 
Sometimes pagination dots are below the fold for tall carousels and/or small displays

**Describe the solution you'd like** 
Would we add an option to allow pagination dots at the top *and*/or the bottom? My current use case would require both top and bottom.

**Comment by am** [comment]
> we've added slots to both pagination and navigation dots

@quinnlangille maybe this should be obvious, but would you mind share how could a custom solution be like using the `slots`? I'm missing how could the setup be like... 

Thank you

**Comment by josephting** [comment]
@am You can edit `Carousel.vue` and move this code block to above `div.VueCarousel-wrapper` and within `section.VueCarousel`.

Copying it will still work properly so you can have both 1 before `.VueCarousel-wrapper` and another after `.VueCarousel-wrapper` to have the pagination slots at both top and bottom of the carousel.

@quinnlangille Would copying the pagination slot to the top and add a new props (perhaps string) OK? Maybe like this.

```
paginationPosition: {
  type: String,
  default: "bottom"
}
```

And perhaps `v-if="paginationEnabled && paginationPosition.indexOf('top') >= 0"` for the top pagination and `v-if="paginationEnabled && paginationPosition.indexOf('bottom') >= 0"` for bottom pagination.

So this will work with either of the below to enable both top and bottom pagination.

```
paginationPosition: 'bottom, top'
paginationPosition: 'top, bottom'
```

**Comment by quinnlangille** [comment]
Yeah, I think that solution could probably work @josephting. We could conditionally render the pagination above or below based on the prop. Good thinking! 

Anyone feel like making a PR for the feature?
